mobile-android-tonic/ssc.config

```diff
@@ -10,6 +10,10 @@ version: v0.0.1
 version_short: 0.0.1
 revision: 1
 
+compile_version: 32
+target_version: 32
+min_sdk: 24
```

Comment:
Generally, we'll want to hide this details as we are committed to certain SDK and NDK level

Reply:
yep, discused with paolo, will revert that.